docs/changelog/changelog_de.md

```diff
@@ -1,5 +1,71 @@
 # Changelog plentyshopPWA
 
+## v1.4.0 (2024-04-08) <a href="https://github.com/plentymarkets/plentyshop-pwa/compare/v1.3.0...v1.4.0" target="_blank" rel="noopener">
+
+
+### Migrations Guide
+
+- Die Upload-Aktion wurde geändert [.github/workflows/upload.yml](https://github.com/plentymarkets/plentyshop-pwa/compare/v1.3.0...v1.4.0#diff-8cf0b5fae548aab9bd49f5020da9c0e35d281984b82b8a5358ffb1c1ae5bec13L5) Aktualisieren Sie die Datei, um die [config-Funktion](https://pwa-docs.plentymarkets.com/guide/setup/deployment#config) zu nutzen.
+- Wir benötigen nun ein Access-Token, um Anfragen an die plentymarkets api zu stellen. [guide](todo)
+- Das Kategorie-Routing wurde aktualisiert und das Präfix /c wurde entfernt. Überprüft, dass keine statischen URLs in Ihrer Anwendung noch /c enthalten.
+  - Um die /c-Routing-Änderung möglich zu machen, wurde die [Kategorieseite](https://github.com/plentymarkets/plentyshop-pwa/compare/v1.3.0...v1.4.0#diff-2f61484eb978aa090fc50dcba90bc44813b45081f25dbff295434cdf6bf219a4) von apps/web/pages/category/[slug].vue nach apps/web/pages/[...slug].vue verschoben.
+
+### Hinzugefügt
+
+- Ein ‘Zurück’-Knopf wurde zur Retouren Form hinzugefügt, um die Navigation zu erleichtern.
+- Tags wurden zu Artikeln und zur Kategorieansichten hinzugefügt.
+- Eine Box-Komponente zur Auswahl von Artikelattributen wurde hinzugefügt.
+- Die Möglichkeit, eine Hauptadresse festzulegen, wurde hinzugefügt.
+- Bildattribute auf Artikelbildern wurden hinzugefügt.
```

Comment:
```suggestion
- Bildattribute auf Artikelbildern
```

Reply:
related pr: https://github.com/plentymarkets/plentyshop-pwa/pull/355. @Tim-M-S

Reply:
I would remove this and add it more clear up there with the box component in 17